src/installer/corehost/cli/hostpolicy/hostpolicy_context.cpp

```diff
@@ -114,13 +114,29 @@ int hostpolicy_context_t::initialize(hostpolicy_init_t &hostpolicy_init, const a
     fx_definition_vector_t::iterator fx_end;
     resolver.get_app_fx_definition_range(&fx_begin, &fx_end);
```

Comment:
This excludes the "App" when in libhost mode - so the loop below would actually skip the first framework.
We may need to tweak the behavior of this helper in correspondence with the loop.